src/main/java/core/basesyntax/service/strategy/strategyimpl/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.strategy.strategyimpl;
+
+import core.basesyntax.dao.Storage;
+import core.basesyntax.model.FruitRecord;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitRecord transaction) {
+        int returnFruits = transaction.getQuantity();
+        Storage.storage.merge(transaction.getFruit(),returnFruits, Integer::sum);
+    }
```

Comment:
Make sure to check the result balance before saving it in the Storage - it should be positive. If the balance is negative, you should throw a RuntimeException to handle this case as per the checklist.